<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_lt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>22</width>
      <height>91</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_rt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>200</y>
      <width>22</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rb">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lb">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_y">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_a">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>460</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_b">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>480</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_right_joy">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_right_joy">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>22</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_left_joy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_left_joy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>22</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Not running</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
